x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.js

```diff
@@ -82,6 +83,12 @@ export class HeadlessChromiumDriverFactory {
         });
 
         page = await browser.newPage();
+
+        // All navigation/waitFor methods default to 30 seconds,
+        // which can cause the job to fail even if we bump timeouts in
+        // the config. Help alleviate errors like
+        // "TimeoutError: waiting for selector ".application" failed: timeout 30000ms exceeded"
```

Comment:
Holy moly... I didn't notice before that the error says 30 seconds. Now that I see that's the case, this PR looks more critical.